fdm-docs/docs/balance/01-nitrogen.md

```diff
@@ -191,13 +191,84 @@ Crop residues (e.g., straw, stover, roots) contain significant amounts of nitrog
+
+    *   **Formula:**
+        `NH3 Emission (kg N / ha) = Application Amount (kg / ha) * TAN Content (fraction) * Emission Factor (fraction)`
+        Where:
+        *   `Application Amount`: `p_app_amount` (kg / ha) - The total amount of fertilizer applied.
+        *   `TAN Content`: `p_nh4_rt` (fraction) - The fraction of total nitrogen that is in ammoniacal form.
+        *   `Emission Factor`: A dimensionless factor representing the proportion of TAN that is volatilized as ammonia. This factor is determined by the application method and the type of land (grassland, cropland, or bare soil) at the time of application.
+
+    *   **Emission Factors for Manure and Compost:**
+        | Application Method    | Grassland | Cropland | Bare Soil |
+        | :-------------------- | :-------- | :------- | :-------- |
+        | Broadcasting          | 0.68      | N/A      | 0.69      |
+        | Narrowband            | 0.264     | 0.36     | 0.36      |
+        | Slotted Coulters      | 0.217     | N/A      | 0.30      |
+        | Shallow Injection     | 0.17      | 0.24     | 0.25      |
+        | Incorporation         | N/A       | N/A      | 0.22      |
+        | Incorporation 2 Tracks| N/A       | N/A      | 0.46      |
+
+        *Note: "N/A" indicates that the method is not typically used or supported for that land type in the calculation model, and will result in an error if attempted.*
+
+*   **Mineral Fertilizers:**
+    For mineral fertilizers, the emission is calculated based on the **total nitrogen content (`p_n_rt`)** of the fertilizer and the **emission factor**.
+
+    *   **Formula:**
+        `NH3 Emission (kg N / ha) = Application Amount (kg / ha) * Total N Content (fraction) * Emission Factor (fraction)`
+        Where:
+        *   `Application Amount`: `p_app_amount` (kg / ha).
+        *   `Total N Content`: `p_n_rt` (fraction).
+        *   `Emission Factor`: `p_ef_nh3` (fraction). This factor can be directly provided in the `FertilizerDetail`. If it is not provided, it is calculated using an empirical formula based on the fertilizer's composition:
+
+        `Emission Factor = p_n_org^2 * K_1 + p_no3_rt * p_s_rt * K_2 + p_nh4_rt^2 * K_3`
+
+        Where:
+        *   `p_n_org`: Organic nitrogen content (calculated as `p_n_rt - p_no3_rt - p_nh4_rt`).
+        *   `p_no3_rt`: Nitrate content.
+        *   `p_nh4_rt`: Ammonium content (TAN).
+        *   `p_s_rt`: Sulfur content.
+        *   `K_1`, `K_2`, `K_3`: Empirical constants.
+            *   If an inhibitor is present: `K_1 = 3.166 * 10^-5`
+            *   If no inhibitor: `K_1 = 7.021 * 10^-5`
+            *   `K_2 = -4.308 * 10^-5`
+            *   `K_3 = 2.498 * 10^-4`
+        *Note: Currently, the presence of an inhibitor (`p_inhibitor`) is hardcoded to `false` in the calculation.*
+
```

Comment:
interface does not allow user to select with/without inhibitor? thats fine, but good to add to action list